script/utils/mento/Contracts.sol

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.18;
+
+import { console } from "forge-std/console.sol";
+
+import { Vm } from "forge-std/Vm.sol";
+import { Chain } from "./Chain.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { IRegistry } from "script/interfaces/IRegistry.sol";
+
+library Contracts {
+  using stdJson for string;
+
+  address private constant REGISTRY_ADDRESS = 0x000000000000000000000000000000000000ce10;
+  // solhint-disable-next-line const-name-snakecase
+  IRegistry private constant registry = IRegistry(REGISTRY_ADDRESS);
+
+  address private constant VM_ADDRESS = address(bytes20(uint160(uint256(keccak256("hevm cheat code")))));
+  // solhint-disable-next-line const-name-snakecase
+  Vm private constant vm = Vm(VM_ADDRESS);
+
+  bytes32 private constant CREATE_HASH = 0x14feaf0665b390ef0561125746780cd06c1876ebed7339648fad78cd5fb754ed;
+
+  struct Cache {
+    mapping(bytes32 => address) contractAddress;
+    bool _dependenciesLoaded;
+    string _dependencies;
+  }
+
+  function loadUpgrade(Cache storage self, string memory upgrade) internal {
+    load(self, string(abi.encodePacked(upgrade, "-00-Create-Proxies")), "latest");
+    load(self, string(abi.encodePacked(upgrade, "-01-Create-Nonupgradeable-Contracts")), "latest");
+    load(self, string(abi.encodePacked(upgrade, "-02-Create-Implementations")), "latest");
+  }
+
+  function load(Cache storage self, string memory script, string memory timestamp) internal {
+    _load(self, script, timestamp, false);
+  }
+
+  function loadSilent(Cache storage self, string memory script, string memory timestamp) internal {
+    _load(self, script, timestamp, true);
+  }
+
+  function _load(Cache storage self, string memory script, string memory timestamp, bool silent) internal {
+    string memory chainId = Chain.idString();
+    string memory root = vm.projectRoot();
+    string memory path = string(
+      abi.encodePacked(root, "/broadcast/", script, ".sol/", chainId, "/", "run-", timestamp, ".json")
+    );
+    string memory json = vm.readFile(path);
+    /**
+     * note(bogdan): Decoding this isn't straightforwad because we're using
```

Comment:
Is this comment still necessary for the newer solidity version?

Reply:
I did not touch the way how we load the contracts so we need to update this function and start using mentioned helper scripts instead to remove the comment